resources/messages_de.properties

```diff
@@ -0,0 +1,193 @@
+ultimatepastebin.accountcredentials.devkey.null=Kein DevKey angegeben.
+
+# Pastes
+ultimatepastebin.paste.untitled=Unbenannt
+ultimatepastebin.paste.key=Schl\u00FCssel: {0}
+ultimatepastebin.paste.url=URL: {0}
+ultimatepastebin.paste.visibility=Sichtbarkeit: {0}
+ultimatepastebin.paste.createddate=Erstellungsdatum: {0}
+ultimatepastebin.paste.expiration=Ablauf: {0}
+ultimatepastebin.paste.expirationdate=Ablaufdatum: {0}
+ultimatepastebin.paste.views=Ansichten: {0}
+ultimatepastebin.paste.highligh=Hervorhebung: {0}
+ultimatepastebin.paste.expiration.never=Nie
+ultimatepastebin.paste.expiration.tenminutes=10 Minuten
+ultimatepastebin.paste.expiration.onehour=1 Stunde
+ultimatepastebin.paste.expiration.oneday=1 Tag
+ultimatepastebin.paste.expiration.oneweek=1 Woche
+ultimatepastebin.paste.expiration.twoweeks=2 Wochen
+ultimatepastebin.paste.expiration.onemonth=1 Monat
+ultimatepastebin.paste.visibility.public=\u00D6ffentlich
+ultimatepastebin.paste.visibility.private=Privat
+ultimatepastebin.paste.visibility.unlisted=Ungelistet
+
+# User information
+ultimatepastebin.user.loggedin={0} Pastes
+ultimatepastebin.user.loggedout=Abgemeldet (Melde dich an, um Pastes zu sehen)
+ultiamtepastebin.user.loginprogress=Meldet an als {0}...
+
+################################
+# Forms
+################################
+
+#Setting form
+ultimatepastebin.settings.form.username=Benutzername:
+ultimatepastebin.settings.form.username.tooltip=Benutzername f\u00FCr die Verbindung mit Pastebin
+ultimatepastebin.settings.form.password=Passwort:
+ultimatepastebin.settings.form.password.tooltip=Passwort f\u00FCr die Verbindung mit Pastebin
+ultimatepastebin.settings.form.devkey=DevKey:
+ultimatepastebin.settings.form.devkey.tooltip=DevKey f\u00FCr Zugriff auf API Funktionen, du musst auf die PasteBin API Seite gehen.
+ultimatepastebin.settings.form.getdevkey=&Erhalte DevKey
+ultimatepastebin.settings.form.getdevkey.tooltip=Leitet dich zu Pastebin weiter, um deinen DevKey zu erhalten
+ultimatepastebin.settings.form.checkaccount=&\u00DCberpr\u00FCfe Benutzerdaten
+ultimatepastebin.settings.form.checkaccount.tooltip=Versuche mit den angegebenen Benutzterdaten mit Pastebin in Verbindung zu treteten
+ultimatepastebin.settings.form.generalsettings=Allgemeine Einstellungen
+ultimatepastebin.settings.form.language=Sprache\:
+ultimatepastebin.settings.form.pastestofetch=Pastes zu laden\:
+ultimatepastebin.settings.form.donate=&Spende, um zu helfen
+ultimatepastebin.settings.form.helptranslate=&Hilf das Plugin zu \u00FCbersetzen\!
+ultimatepastebin.settings.form.language.help=Sprache des Plugins
+ultimatepastebin.settings.form.pastestofetch.help=Maximale Anzahl an Pastes, die f\u00FCr den Benutzer geladen werden (1-100)
+ultimatepastebin.settings.form.donation.help=Wenn dieses Plugin dir hilft kannst du mir mit einer Spende helfen \:D
+ultimatepastebin.settings.form.helptranslate.help=Hilf bei der \u00DCbersetzung in eine neue Sprache oder verbessere eine \u00DVbersetzung\!
```

Comment:
@minekrexx Checking your translation and it's not working due to a error on this line, you typed a wrong unicode value "\u00DV", the "V" is invalid, i can imagine that probably would be "C" instead due to a proximity, but since i know nothing about german and can't confirm it, could you review please?